DEPS

```diff
@@ -17,7 +17,7 @@ deps = {
   },
   "vendor/bat-native-tweetnacl": "https://github.com/brave-intl/bat-native-tweetnacl.git@8b424ccf29957fe01c88c36076fd32006a357887",
   "vendor/gn-project-generators": "https://github.com/brave/gn-project-generators.git@b76e14b162aa0ce40f11920ec94bfc12da29e5d0",
-  "vendor/web-discovery-project": "https://github.com/brave/web-discovery-project@636280b047fc54999a5e632f2e23671c3048f1e0",
+  "vendor/web-discovery-project": "https://github.com/brave/web-discovery-project@6d72992d2236db3cfd5cd1d2af4c13267780cd63",
```

Comment:
needs updating once https://github.com/brave/web-discovery-project/pull/435 is merged